File: iSeeCars.Business/Managers/BrandManager.cs
using iSeeCars.Core.Entities;

namespace iSeeCars.Business.Managers
{
    public class BrandManager
    {
        List<Brand> brands = new List<Brand>();

        public void AddBrand(Brand brand)
        {
            brands.Add(brand);
        }

        public List<Brand> GetAllBrands()
        {
            return brands;
        }
        public void RemoveBrand(Brand brand)
        {
            brands.Remove(brand);
        }
    }
}

File: iSeeCars.java/src/main/java/iseecars/manager/BrandManager.java
package iseecars.manager;

import iseecars.model.Brand;

import java.util.ArrayList;
import java.util.List;

public class BrandManager {
    private List<Brand> brands = new ArrayList<Brand>();

    public void addBrand(Brand brand){
        brands.add(brand);
    }
    public List<Brand> getAllBrands(){
        return brands;
    }
    public void removeBrand(Brand brand){
        brands.remove(brand);
    }
}
